subsys/bluetooth/audio/shell/bap_broadcast_assistant.c

```diff
@@ -900,6 +905,24 @@ static int cmd_bap_broadcast_assistant_add_pa_sync(const struct shell *sh,
 		}
 	}
 
+	/* use the BASE to verify the BIS indexes set by command */
+	for (size_t j = 0U; j < param.num_subgroups; j++) {
+		if (bis_bitfield_req == 0) {
+			/* Request a PA sync without BIS sync */
+			subgroup_params[j].bis_sync = 0;
+		} else {
+			subgroups_bis_sync |= subgroup_params[j].bis_sync;
+		}
```

Comment:
Isn't this missing? 

```suggestion
		} else {
			subgroups_bis_sync |= subgroup_params[j].bis_sync;
			subgroup_params[j].bis_sync &= bis_bitfield_req;
		}
```

Otherwise I don't see how else the `bis_bitfield_req` will affect the parameters